index.html

```diff
@@ -4633,21 +4633,139 @@ <h2>Key and Signature Expiration</h2>
     </section>
 
     <section>
-      <h2>Key Revocation and Recovery</h2>
 
+      <h2>Verification Method Rotation</h2>
+  
+      <p>Verification method rotation is a proactive security measure.</p>
+  
       <p>
-It is up to <a>DID methods</a> to define how revocation of cryptographic keys
-might occur. <a>DID method</a> specifications are also expected
-to enable support for a quorum of trusted parties to enable key recovery. Some
-of the facilities to do so are suggested in Section
-<a href="#did-controller"></a>. Not all <a>DID method</a>
-specifications will recognize control from <a>DIDs</a> registered using other
-<a>DID methods</a> and they might restrict third-party control to <a>DIDs</a>
-that use the same method. Access control and key recovery in a
-<a>DID method</a> specification can also include a time lock feature to
-protect against key compromise by maintaining a second track of control for
-recovery.
-</p>
+        Verification method rotation applies only to the current or latest
+        version of a DID Document.
+      </p>
+  
+      <p>
+        When a verification method has been active for a long time, or used
+        for many operations, a controller might wish to perform a rotation.
+      </p>
+  
+      <p class="advisement">
+        It is considered a best practice to perform verification method
+        rotation on a regular basis.
+      </p>
+
+
+      <p class="advisement">
+        Proofs or signatures that rely on verification methods that are not
+        present in the latest version of a DID Document are not impacted by
+        rotation, and may require additional information to mitigate compromise.
+      </p>
+  
+      <p>
+        Section <a href="#method-operations"></a> specifies the
+        <a>DID</a> operations to be supported by a
+        <a>DID method</a> specification, including
+        <a href="#update">update</a>
+        which MUST be used to perform a verification method rotation.
```

Comment:
```suggestion
        which is expected to be used to perform a verification method rotation.
```

The Security Considerations section is non-normative, we can't put normative language in it.